zookeeper-docs/src/main/resources/markdown/zookeeperAdmin.md

```diff
@@ -1142,6 +1142,10 @@ property, when available, is noted below.
     The intent is for the clients not to retry them immediately.
     When set to 0, no requests will be throttled. The default is 0.
 
+* *leader.closeSocketAsync*
+   (Java system property only: **leader.closeSocketAsync**)
```

Comment:
Please also tag this property with the ZooKeeper version (as you can see in other properties added recently). Something like`**New in 3.6.2:**` 

I think we should backport this also to branch 3.6, as it is a good optional feature for those who use SSL. @eolivelli do you agree?

Reply:
done.